### ISSUE TITLE
Make cost allocation tags work with Athena

### ISSUE THREAD

**Issue by tmclaugh** [ServerlessOpsIO/ApplicationCostMonitoring#2]
Tried to use cost allocation tags but they appear to be causing this issue with Athena.

```
Your query has the following error(s):

HIVE_INVALID_METADATA: com.facebook.presto.hive.DataCatalogException: Error: : expected at the position 10 of 'struct<aws\:cloudformation\:stack-name:string,aws\:createdBy:string,user\:STAGE:string>' but '\' is found. (Service: null; Status Code: 0; Error Code: null; Request ID: null)

This query ran against the "aws_adm1" database, unless qualified by the query. Please post the error message on our forum or contact customer support with Query Id: d41e1bac-8ef9-4d2a-be4c-c73afb8093b1.
```

Have noticed that those tags are not present on all items.  I think this may be causing the issue.  Though, they are recorded as nulls in the item document.  Maybe Athena can't handle these nulls?  Not sure.  Have disabled cost allocation tags in account for now.

**Comment by tmclaugh** [comment]
Still seeing these tags.... Should just drop them from the S3 writer.

**Comment by tmclaugh** [comment]
Fixed after 0fd4ef872c3db063c312a32560270099fa35b233.

Still issues with schema changes.  Doing to handle that a different repo when I decompose this application.

**Comment by tmclaugh** [comment]
Must set SCHEMA_CHANGE_HANDLING now.
